examples/jsm/objects/ReflectorForSSRPass.js

```diff
@@ -230,7 +230,9 @@ var Reflector = function ( geometry, options ) {
 		renderer.state.buffers.depth.setMask( true ); // make sure the depth buffer is writable so it can be properly cleared, see #18897
 
 		if ( renderer.autoClear === false ) renderer.clear();
+		// this.visible = false;
 		renderer.render( scene, virtualCamera );
+		// this.visible = true;
```

Comment:
@gonnavis you're okay with leaving that commented code here?

Reply:
Oh sorry, I'm just okay with the logic. I do feel it is not good to leave comments here, but I am afraid of making too many opinions, thinking
 it may can be solved in the process of merge.

@ycw Could you please remove these comments?

Reply:
removed

Reply:
Hello @mrdoob , I think should be okay now.